_layouts/newsletters.md

```diff
@@ -0,0 +1,45 @@
+---
+type: pages
+layout: default
+---
+<link rel="stylesheet" href="/assets/css/main.css">
+
+<div class="localization">
+  <a href="/en/newsletters/">en</a>
+  {% for lang in site.languages %}
+    | <a href="/{{ lang }}/newsletters/">{{lang}}</a>
+  {% endfor %}
+</div>
+
+<h1 class="post-title">Newsletters</h1>
+
+{%- if page.lang == "en" -%}
+  {% include newsletter-signup.html %}
+{%- endif -%}
+
+<div class="post-content">
+  {{ content }}
+</div>
```

Comment:
Nitpick: the English page is devoid of content (which is fine), so this renders an empty div on that page.  That doesn't really matter, but it looks odd to me.  Maybe consider gating this with an `{% if content != ""%}`

Reply:
done